core/commands/remote.py

```diff
@@ -1,39 +1,39 @@
 import sublime
-from sublime_plugin import TextCommand
+from sublime_plugin import WindowCommand
 
 from ..git_command import GitCommand
 from ...common import util
 from ..ui_mixins.quick_panel import show_remote_panel
 
 
-class GsRemoteAddCommand(TextCommand, GitCommand):
+class GsRemoteAddCommand(WindowCommand, GitCommand):
     """
     Add remotes
     """
 
-    def run(self, edit):
+    def run(self):
```

Comment:
Should WindowCommand not take an edit, 
I tested and removed it

Reply:
You are right. 👍